### PR TITLE
change peer.address logrus representaion.

### PR DESCRIPTION
**Pull request by altaastro**

### CODE REVIEW

**Comment by altaastro** [comment]
The opentracing specification describes "peer.address" logger tag as "ip:port"-a-like string, but current middleware realization writes to log a jsonpb.Marrshal'ed net.TCPAddr structure (net.Addr interface). So, perhaps, it is better to call net.Addr's .String()before passing into tag.Set() method.

**Comment by altaastro** [comment]
q

**Comment by altaastro** [comment]
I've signed CLA

**Comment by altaastro** [comment]
Here's link to opentracing spec https://github.com/opentracing/specification/blob/master/semantic_conventions.md

**Comment by altaastro** [comment]
Hello again. Can I have some feedback, please? Our company's project is using this middleware, but current
grpc-logrus doesn't follow the opentracings specs (in peer.Address), and because of that those logs are hard to parse with elastic.

**Comment by domgreen** [comment]
My only concern would be if someone is already using the current representation but I also agree that moving to the standard is a good idea.
In #115 I am adding a representation of what is logged and the given types ... if we pull this after that I suggest we update those docs as well.

@yifanzz @mwitkow thoughts?

**Comment by mwitkow** [comment]
Yea, legit. Sorry about that it was just a massive oversight.
